index.html

```diff
@@ -45,7 +45,7 @@
 
 <body>
 <div id="abstract">
-  <p>This document describes the best practices for identifying language and base direction for strings used on the Web.</p>
+  <p>This document describes the best practices for identifying language and paragraph direction for strings used on the Web.</p>
```

Comment:
Why was this changed to "paragraph direction"?

Reply:
If we decide to use "paragraph direction", we might want to mention it in https://www.w3.org/International/articles/lang-bidi-use-cases/index.en.html#definitions too.